components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -13,21 +19,19 @@ ProgressIndicator.defaultProps = {
 };
 
 export default function ProgressIndicator({ stepNumber, totalSteps }) {
-  if (totalSteps < 1) return null;
-
-  let currentStep = stepNumber;
-
-  if (currentStep < 0) currentStep = 0;
-  if (currentStep > totalSteps) currentStep = totalSteps;
+  if (totalSteps === 0) return null;
```

Comment:
You could drop this line, make `developmentErrors.totalStepsTooLow` say: `'"totalSteps" must be greater than zero'`.